src/title/TitleStream.js

```diff
@@ -23,8 +23,8 @@ type Props = $ReadOnly<{|
   ...SelectorProps,
 |}>;
 
-class TitleStream extends PureComponent<Props> {
-  styles = createStyleSheet({
+const TitleStream = (props: Props) => {
+  const titleStreamStyles = createStyleSheet({
```

Comment:
nit: This gets renamed to `titleStreamStyles` in the first commit, then `componentStyles` in the second. It'd make the diffs slightly cleaner to read if it becomes `componentStyles` directly in the first commit.

I'd make this change with `git rebase -i`, choose `edit` (or `e`) for the first commit, let it run, make the edit, say `git add -u && git rebase --continue` (actually I [have an alias](https://gist.github.com/gnprice/5f71b63464a76bd7e0480f207b1bbff3) `git rac` to save typing that), probably resolve conflict which probably just means picking the appropriate lines from what's there, say `git rac` again. Takes about a minute.

This example isn't a critical thing to fix, but it is a real (though small) cleanup and is an easy edit to make, as edits to commit series go. I think it's worth the practice to do easy ones like this -- there will be plenty of other cases where it's trickier :slightly_smiling_face:, and it's good to get to where the mechanics of it are so familiar they're easy and almost automatic to do.